src/kritter/ktextclient.py

```diff
@@ -1,22 +1,32 @@
-class KtextClient:
-    def __init__(self):
-        self.callback_receive = None
+#
+# This file is part of Vizy 
+#
+# All Vizy source code is provided under the terms of the
+# GNU General Public License v2 (http://www.gnu.org/licenses/gpl-2.0.html).
+# Those wishing to use Vizy source code, software and/or
+# technologies under different licensing terms should contact us at
+# support@charmedlabs.com. 
+#
+from collections import UserString
 
-    def text(self, to, text):
-        pass
+class Image:
+    def __init__(self, image):
+        self.image = image
 
-    def image(self, to, image):
-        pass
+class KtextClient:
```

Comment:
I wanted to change the vizy class TelegramDialog to get generic.  This class is tied to Telegram, so it stays as TelegramClient.  Please change back.